### DISCUSSION TITLE
[solved] How sort by latest post?

### DISCUSSION THREAD

**Discussion by twMat** [community/community#3142] [Discussions] [answered]
Edit: I realized my question was a misunderstanding: Threads are automatically sorted by latest post. I guess my question now is "How do I delete this question here" because it's just embarrassing to have it remain.

Original question below:
---
Either the feature is lacking or it is just not visible enough but how do I sort discussions by latest post, like is possible for e.g Issues?

**Comment by PranamBhat** [comment, marked as answer]
Hello @twMat 

Don't worry!

Just select my comment as "**Answer Selected**" and leave.

It will be sorted as "**Answered**". Thank you!
